docs/operation/cluster_and_service_configuration.rst

```diff
@@ -0,0 +1,64 @@
+Cluster and Services Configurations
+===================================
+
+This section contains information describing the list of available Cluster and
+Services Configurations, the :term:`Namespace` to which they belong,
+the :term:`ConfigMap` where these configurations are store and finally the
+different ways of customizing and applying any given Cluster and Services
+Configurations.
+
+Managing Cluster and Services Configurations
+********************************************
```

Comment:
Why using '***' an then '---' for titles? It will create two different levels of titles, which does not make real sense here as we only have two :) 
I recommend changing this one with '-----'

Reply:
ok